apps/predictions/lib/prediction.ex

```diff
@@ -22,10 +23,15 @@ defmodule Predictions.Prediction do
 
   @type id_t :: String.t()
   @type schedule_relationship :: nil | :added | :unscheduled | :cancelled | :skipped | :no_data
+
+  @typedoc "If the predicted stop has a parent stop (station), then that *may* be the stop set on the `stop` field. In some cases, it may contain a platform stop when the stop has a parent. The unmodified stop_id for the prediction can be found in the platform_stop_id field."
```

Comment:
question: Is this saying that the stop may sometimes be the parent station, but sometimes isn't?